CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Slow rendering performance with HiDPI displays, especially on macOS
 - Keys swallowed during search when pressing them right before releasing backspace
 - Crash when a wrapped line is rotated into the last line
+- Yellow/Red colors for message bar not being picked from user's config
```

Comment:
Maybe this should be in the `Changed` section, since this was never intended to be this way.

Reply:
I guess it could be there.